ksp_plugin_adapter/flight_planner.cs

```diff
@@ -684,8 +710,31 @@ class FlightPlanner : VesselSupervisedWindowRenderer {
     }
   }
 
+  private void UpdateAdaptiveStepParameters(
+      ref FlightPlanAdaptiveStepParameters parameters) {
+    parameters.length_integration_tolerance =
+        integration_tolerances_[length_integration_tolerance_index_];
+    parameters.speed_integration_tolerance =
+        integration_tolerances_[speed_integration_tolerance_index_];
+    parameters.max_steps = max_steps_[max_steps_index_];
+  }
+
   private IntPtr plugin => adapter_.Plugin();
 
+  private static readonly double[] integration_tolerances_ =
+      {1e-6, 1e-5, 1e-4, 1e-3, 1e-2, 1e-1, 1e0, 1e1, 1e2, 1e3, 1e4, 1e5, 1e6};
+  private static readonly string[] length_integration_tolerances_names_ = {
+      "1 µm", "10 µm", "100 µm", "1 mm", "1 cm", "10 cm", "1 m", "10 m",
+      "100 m", "1 km", "10 km", "100 km", "1000 km"
+  };
+  private static readonly string[] speed_integration_tolerances_names_ = {
+      "1 µm/s", "10 µm/s", "100 µm/s", "1 mm/s", "1 cm/s", "10 cm/s", "1 m/s",
+      "10 m/s", "100 m/s", "1 km/s", "10 km/s", "100 km/s", "1000 km/s"
+  };
```

Comment:
Unused, remove.
Also remove this TODO  in the main window :
https://github.com/mockingbirdnest/Principia/blob/5c17d2580bcebc754f9e6e72dae3e71059547ec5/ksp_plugin_adapter/main_window.cs#L430
Let’s not go there.